### DISCUSSION TITLE
/tmp getting full after running only a few playbooks

### DISCUSSION THREAD

**Discussion by smitch35** [semaphoreui/semaphore#1422] [Q&A]
I was wondering if other users are seeing this and what they are doing.  Basically our /tmp folder is getting full of stuff from the playbook runs we do to the point we have a cronjob to clean that area out each weekend.  

Did we miss a step in the semaphore configuration to have it do this for us?  Is this not an issue for others?

**Comment by ansibleguy** [comment]
Greetings.

This might be a duplicate of https://github.com/ansible-semaphore/semaphore/pull/1134
If it is so - please comment there and close this issue.

**Comment by smitch35** [comment]
closing as it does appear related to #1134
